src/next_rebase/main.py

```diff
@@ -86,6 +91,15 @@ async def update_info():
     for guild in client.guilds:
         guser = guild.get_member(client.user.id)
         await guser.edit(nick=f'Rebase In: {int(hours)}h {int(minutes)}m ')
+
+        if int(minutes) <= 1:
+            webhook = discord.Webhook.from_url(
+                os.environ["DISCORD_REBASE_BOT_WEBHOOK_URL"],
+                adapter=discord.RequestsWebhookAdapter()
+            )
+            webhook.send(
+                "Rebasing momentarily, @Klimates! (:deciduous_tree:, :deciduous_tree:)"
```

Comment:
Yeah I don't think tagging all Klimates is a good idea... I think just sending this message to #general without an @ mention will be fine

Reply:
Nice, thanks  for the guidance 😊